<launch>
  <group ns="hog">
   <param name ="/use_sim_time" value="true"/>
   <node name="rplidarNode0"          pkg="rplidar_ros"  type="rplidarNode" args="scan:=scan0" output="screen">
          <param name="serial_port"         type="string" value="/dev/ttyUSB0"/>
          <param name="serial_baudrate"     type="int"    value="256000"/><!--A3 -->
          <param name="frame_id"            type="string" value="laser_hog"/>
          <param name="inverted"            type="bool"   value="false"/>
	  <param name="angle_compensate"    type="bool"   value="true"/>
	  <param name="scan_mode"           type="string" value="Sensitivity"/>
   </node>
   <param name ="/use_sim_time" value="true"/>
   <node pkg="tf2_ros" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 1 map laser_hog" />
 </group>
</launch>

